### ISSUE TITLE
Warning message about deprecated Docker API call

### ISSUE THREAD

**Issue by yacchin1205** [jupyter/tmpnb#219]
The Docker daemon shows the warning message below when tmpnb starts a container.

```
WARN[6322] DEPRECATED: Setting host configuration options when the container starts is deprecated and will be removed in Docker 1.12
```

Though `port_bindings` parameter is passed to `docker_client.start` (https://github.com/jupyter/tmpnb/blob/master/dockworker.py#L166-L168), 
according to [docker-py docs](https://docker-py.readthedocs.org/en/latest/api/#start),
I assume that it should be put to `host_config` in `docker_client.create_container` (https://github.com/jupyter/tmpnb/blob/master/dockworker.py#L150-L157).

**Comment by rgbkrk** [comment]
Fixed in #220, thanks again @yacchin1205!

**Comment by yacchin1205** [comment]
Thanks, @rgbkrk !
